bigtop-packages/src/common/bigtop-select/distro-select

```diff
@@ -154,7 +154,7 @@ def listPackages( packages ):
   if packages == None:
     packages = leaves
 
-  packages.sort()
+  sorted(packages)
```

Comment:
The `sorted` function doesn't change the original list, so these lines seem to have no effect.

Reply:
> The `sorted` function doesn't change the original list, so these lines seem to have no effect.

Thank you for your feedback. It has been modified.